cmd/artifact/command/push.go

```diff
@@ -47,8 +48,8 @@ func pushCommand(options *Options) *cobra.Command {
 			return nil
 		},
 	}
-	command.Flags().StringVar(&releaseManagerClient.BaseURL, "http-base-url", "https://release-manager.dev.lunarway.com", "address of the http release manager server")
-	command.Flags().StringVar(&releaseManagerClient.Metadata.AuthToken, "http-auth-token", "", "auth token for the http service")
+	command.Flags().StringVar(&releaseManagerClient.BaseURL, "http-base-url", os.Getenv("HAMCTL_URL"), "address of the http release manager server")
+	command.Flags().StringVar(&releaseManagerClient.Metadata.AuthToken, "http-auth-token", os.Getenv("HAMCTL_AUTH_TOKEN"), "auth token for the http service")
```

Comment:
This will leak thr token to the console if you use --help.

I know it is done this way in hamct as well, but I'd rather we found a way to avoid it, and instead wrote `from HAMCTL_..` in the value.

Reply:
That makes sense. What about setting the value(s) based on the the arg or os.GetEnv functions and leave the default value empty here? I would expect arg to take precedence if it is set?

Reply:
I think leaving the default to be empty fine, then just let the arg take precedence, check the env var, and if none is set, break out?

Reply:
Agree. There is a trick somewhere that just travers the flag set and checks if an env variable is available. That would be pretty nice so all flags can be set by env variables or a setting.

Pseudo coded like this:

```go
for _, flag := range command.Flags() {
  envVar := fmt.Sprintf("HAMCTL_%s",  strings.Replace(flag.Name(), "-", "_"))
  v, ok := os.LookupEnv(envVar)
  if !ok {
    continue
  }
  flag.Value(v)
}
```

The above makes no sense in practice regarding code. Just to show how it could work.